Packs/CrowdStrikeOpenAPI/Integrations/CrowdStrikeOpenAPI/CrowdStrikeOpenAPI.py

```diff
@@ -9575,7 +9575,7 @@ def main():
     demisto.debug(f'Command being called is {command}')
 
     try:
-        requests.packages.urllib3.disable_warnings()
+        requests.packages.urllib3.disable_warnings()  # type: ignore
```

Comment:
No need to ignore this, just import the urllib3 and remove the requests prefix
```suggestion
        urllib3.disable_warnings()  # type: ignore
```